### ISSUE TITLE
Experiment Repository Creation Request for Basics of HTML

### ISSUE THREAD

**Issue by vlabsdev-iitd** [virtual-labs/engineers-forum#658]
## *Repository Creation Request for Basics of HTML*

1. #### Coordinating Institute: IIT Delhi

2. #### Approver’s Name: Dr. Ranjan Bose

3. #### Approved Proposal: [HTML proposal comments.pdf](https://github.com/virtual-labs/engineers-forum/files/5847742/HTML.proposal.comments.pdf)

4. #### Primary GitHub Handle Details: 
   ##### Name: Virtual Labs IIT Delhi
   ##### GitHub Handle: vlabsdev-iitd
   ##### Email id: vlabsdev.iitd@gmail.com

5. #### Secondary GitHub Handle Details: 
    ##### Name: Sanjeet Kumar
    ##### GitHub Handle: sanjeetkumar
    ##### Email id: sanjeetkumar.iitd@gmail.com

    ##### Name: Tanmay Tarun Das
    ##### GitHub Handle: Tanmayiitd
    ##### Email id: ttdas.iitd@gmail.com

    ##### Name: Prateek Sharma
    ##### GitHub Handle: prateek94iitd
    ##### Email id: prateek94.iitd@gmail.com
 
    ##### Name: Akshat Agarwal
    ##### GitHub Handle: akshat-vlabsdeviitd
    ##### Email id: akshat.vlabs.iit@gmail.com

**Comment by pavanchow** [comment]
@vlabsdev-iitd   Please find the links for **Basics of HTML** experiments and populate the repositories with sources of the experiments as detailed in Step 4 of the  [migration document](https://github.com/virtual-labs/engineers-forum/blob/master/migration-process.md).


1. Introduction to HTML - https://github.com/virtual-labs/exp-introduction-to-html-iitd

2. Applying Attributes in HTML Tags - https://github.com/virtual-labs/exp-html-tags-iitd

3. Inserting Images through <img> tags - https://github.com/virtual-labs/exp-img-tags-in-html-iitd

4. Using Anchor Tags for Hyperlinks - https://github.com/virtual-labs/exp-anchor-tags-in-html-iitd

5. How <marquee> Tag works in HTML - https://github.com/virtual-labs/exp-marquee-tags-in-html-iitd

6. Creating Tables in HTML - https://github.com/virtual-labs/exp-tables-in-html-iitd

7. Types of Lists in HTML - https://github.com/virtual-labs/exp-lists-in-html-iitd

8. Working of \<div> Tag in HTML - https://github.com/virtual-labs/exp-div-tag-in-html-iitd

9. Embedding through <iframe> Tag - https://github.com/virtual-labs/exp-iframe-tag-in-html-iitd

10. Creating Web Page Layout in HTML - https://github.com/virtual-labs/exp-webpage-layout-in-html-iitd

**Comment by pavanchow** [comment]
@vlabsdev-iitd @sanjeetkumar @Tanmayiitd @prateek94iitd @akshat-vlabsiitd
Please click on each link and accept invitations to push code to **Basics of HTML** exp repositories.

1. Introduction to HTML - https://github.com/virtual-labs/exp-introduction-to-html-iitd/invitations

2. Applying Attributes in HTML Tags - https://github.com/virtual-labs/exp-html-tags-iitd/invitations

3. Inserting Images through <img> tags - https://github.com/virtual-labs/exp-img-tags-in-html-iitd/invitations

4. Using Anchor Tags for Hyperlinks - https://github.com/virtual-labs/exp-anchor-tags-in-html-iitd/invitations

5. How <marquee> Tag works in HTML - https://github.com/virtual-labs/exp-marquee-tags-in-html-iitd/invitations

6. Creating Tables in HTML - https://github.com/virtual-labs/exp-tables-in-html-iitd/invitations

7. Types of Lists in HTML - https://github.com/virtual-labs/exp-lists-in-html-iitd/invitations

8. Working of \<div> Tag in HTML - https://github.com/virtual-labs/exp-div-tag-in-html-iitd/invitations

9. Embedding through <iframe> Tag - https://github.com/virtual-labs/exp-iframe-tag-in-html-iitd/invitations

10. Creating Web Page Layout in HTML - https://github.com/virtual-labs/exp-webpage-layout-in-html-iitd/invitations

**Comment by vlabsdev-iitd** [comment]
Initial Version of Experiment 1 is released with tagging v1.0.0, kindly check whether everything is proper and in place.

**Comment by pavanchow** [comment]
@vlabsdev-iitd Please click on the link which is right under the about section of your experiment to check your release.

![image](https://user-images.githubusercontent.com/24794698/106374373-f4581180-63a8-11eb-9263-fb98c7705336.png)

**^ Same process for every experiment

**Comment by vlabsdev-iitd** [comment]
Checked the release, everything is working properly. What will be the next step, shall we begin uploading other experiments.

**Comment by pavanchow** [comment]
@vlabsdev-iitd Yes, please upload sources to all other experiments and comment on #585 for rehosting lab.

**Comment by vlabsdev-iitd** [comment]
@pavanchow all experiments uploaded, tested and tagged.

**Comment by pavanchow** [comment]
@vlabsdev-iitd Please create tags for the following experiments to host the lab, 

1. https://github.com/virtual-labs/exp-img-tags-in-html-iitd/tags
2. https://github.com/virtual-labs/exp-marquee-tags-in-html-iitd/tags

**Comment by vlabsdev-iitd** [comment]
@pavanchow  tagging for above mentioned experiments done, kindly check and re host the experiments.

**Comment by pavanchow** [comment]
@vlabsdev-iitd Please follow this issue https://github.com/virtual-labs/engineers-forum/issues/585#issuecomment-778937502 process for rehosting request.
